src/api/store/userprofile.py

```diff
@@ -371,9 +371,16 @@ def create_user(self, context: Context, args) -> Tuple[dict, MassEnergizeAPIErro
       # added for special case of guest users, mark them as such in user_info
       user_info = args.get('user_info', None)     
       full_name = args.get('full_name')
+      preferred_name = args.get('preferred_name', None)
+
+      new_user_type = STANDARD_USER
       if 'guest' in full_name.lower():
```

Comment:
@BradHN1  let me add `is_guest:true` to the api request when I am hitting the `users.create` route. Instead of using the name to differentiate standard user from guest user yeah? 
You can expect `is_guest` in the handler, but dont make it compulsory.